app/views/categories/_equipment_models_table.html.erb

```diff
@@ -0,0 +1,40 @@
+<table id="table_woo" class="datatable table table-striped table-bordered">
+  <thead>
+  <tr>
+    <th>Name</th>
+    <th class="no_sort">Current Status</th>
+    <th>Available</th>
+    <th>Reserved</th>
+    <th>Checked-Out</th>
+    <th>Overdue</th>
+    <th>Total Stock</th>
+  <% if can? :manage, EquipmentModel %>
+    <th class="no_sort"> </th>
+    <th class="no_sort"> </th>
+  <% end %>
+  </tr>
+  </thead>
+  <tbody>
+  <% for equipment_model in @category.equipment_models %>
+    <tr>
+      <td><%= link_to equipment_model.name, equipment_model %></td>
+      <% if equipment_model.availability(Date.today) == 'all' %>
+        <td><span class="label label-success eo-table">All Available</span></td>
+      <% elsif equipment_model.availability(Date.today) == 'some' %>
+        <td><span class="label label-warning eo-table">Some Available</span></td>
+      <% else %>
+        <td><span class="label label-danger eo-table">None Available</span></td>
+      <% end %>
+      <td><%= equipment_model.available_count(Date.today) %></td>
+      <td><%= equipment_model.num_reserved(Date.today, Date.today, equipment_model.reservations)%></td>
+      <td><%= equipment_model.reservations.checked_out.count %></td>
```

Comment:
This is causing an N+1 query since it's checking for the reservations of each equipment model separately. You should add `includes(:reservations)` to the query above (i.e. `@category.equipment_models.includes(:reservations)`)